src/ActionModal/components/ActionModal.vue

```diff
@@ -477,7 +477,6 @@ export default {
         userAddress: this.session.address,
         rewards: this.rewards,
         totalRewards: this.overview.totalRewards,
-        delegations: this.delegations,
```

Comment:
how come this is no longer needed?

Reply:
I didn't find any use of delegations in the context. 5 withdraws are calculated based on the rewards.